azure-pipelines.vrt-baseline.yml

```diff
@@ -81,17 +73,13 @@ jobs:
           vrTestPackageName: '@fluentui/docs'
           vrTestPackagePath: 'packages/fluentui/docs'
 
-      - bash: node node_modules/vrscreenshotdiff/lib/index.js release --clientType "FluentUI-v0" --buildId $(Build.BuildId)
-        displayName: 'Run Screenshotdiff update baseline'
+      - bash: vr-app run-diff --buildType release --screenshotsDirectory ./screenshots --clientType "FLUENTUI" --locationPrefix 'FluentUI-v0' --locationPostfix 'vrscreenshotv0' --pipelineId $(pipelineId)
+        displayName: 'Run fluentui-screenshotdiff'
         env:
-          API_URL: $(System.CollectionUri)
           API_TOKEN: $(fabric-public-pipeline-access-PAT)
-          API_REPOSITORY: $(Build.Repository.Name)
-          API_PROJECT: $(System.TeamProject)
-          SCREENSHOT_ARTIFACT_FOLDER: vrscreenshotv0
           GITHUB_API_TOKEN: $(githubRepoStatusPAT)
-          STORAGE_ACCOUNT_FLUENTUI: $(STORAGE-ACCOUNT-FLUENTUI)
-          STORAGE_KEY_FLUENTUI: $(STORAGE-KEY-BLOB-FLUENTUI)
-          BLOB_CONNECTION_STRING: $(BLOB-CONNECTION-STRING)
-          VR_APPROVAL_CLIENT_SECRET: $(VR-APPROVAL-CLIENT-SECRET)
-          VR_APPROVAL_HOST: $(VR_APPROVAL_HOST)
+          STORAGE_CONNECTION_STRING:  $(BLOB-CONNECTION-STRING)
+          VR_APP_CLIENT_SECRET: $(VR-APPROVAL-CLIENT-SECRET)
+          VR_APP_CLIENT_ID: '288a69b6-760d-4c1f-ad6d-0183b5e5740f'
+          VR_APP_API_URL: 'https://vrapprovalfunction-prod.azurewebsites.net/api/'
```

Comment:
Are there possible security concerns with having these explicitly written out? Should `VR_APP_CLIENT_ID` and `VR_APP_API_URL` be stored as pipeline variables instead

Reply:
yes...adding them